osm2streets/src/render.rs

```diff
@@ -240,6 +251,65 @@ impl StreetNetwork {
         let output = serde_json::to_string_pretty(&obj)?;
         Ok(output)
     }
+
+    /// For an intersection, show all the movements.
+    pub fn debug_movements_geojson(&self) -> Result<String> {
+        let arrow_shift_dist = if self.config.driving_side == DrivingSide::Right {
```

Comment:
```suggestion
        // Multiple arrows with endpoints on one road would overlap and look cluttered. Offset them to help.
        let arrow_shift_dist = if self.config.driving_side == DrivingSide::Right {
```

I had to play around with this param to understand the purpose